lib/elixir/lib/port.ex

```diff
@@ -88,10 +123,13 @@ defmodule Port do
   end
 
   @doc """
-  See [`:erlang.ports/0`](http://www.erlang.org/doc/man/erlang.html#ports-0)
+  Returns a list of the ports of the current node.
```

Comment:
I'm not too sure about this one, but wouldn't it be "list of ports __for__ the current node" ?